includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -705,7 +705,11 @@ public function get_latest_charge_from_intent( $intent ) {
 		if ( ! empty( $intent->charges->data ) ) {
 			return end( $intent->charges->data );
 		} else {
-			return $this->get_charge_object( $intent->latest_charge );
+			try {
+				return $this->get_charge_object( $intent->latest_charge );
+			} catch ( WC_Stripe_Exception $e ) {
```

Comment:
If we don't let this bubble up, the following flows that expect this exception will never be triggered:
- WC_Stripe_UPE_Payment_Gateway::process_payment_with_saved_payment_method * → save_intent_to_order
- WC_Stripe_UPE_Payment_Gateway::process_upe_redirect_payment * → process_order_for_confirmed_intent → save_intent_to_order
- WC_Stripe_Intent_Controller::update_order_status_ajax * → process_order_for_confirmed_intent → save_intent_to_order
- WC_Gateway_Stripe::process_payment * → create_intent → save_intent_to_order
- WC_Stripe_Intent_Controller::update_failed_order_ajax * → save_intent_to_order
- WC_Stripe_Pre_Orders_Trait::process_pre_order_release_payment * → create_and_confirm_intent_for_off_session → save_intent_to_order